packages/dd-trace/test/appsec/iast/overhead-controller.spec.js

```diff
@@ -0,0 +1,112 @@
+const overheadController = require('../../../src/appsec/iast/overhead-controller')
+
+describe('Overhead controller', () => {
+  const oceContextKey = overheadController.OVERHEAD_CONTROLLER_CONTEXT_KEY
+
+  describe('Initialize OCE context', () => {
+    it('should populate oce context', () => {
+      const iastContext = {}
+      overheadController.initializeRequestContext(iastContext)
+      expect(iastContext).to.have.nested.property(overheadController.OVERHEAD_CONTROLLER_CONTEXT_KEY)
+    })
+  })
+
+  describe('Single shot operation', () => {
+    it('should not allow with no context', () => {
+      expect(overheadController.hasQuotaSingleShot(overheadController.SINGLE_SHOT_OPERATIONS[0])).to.be.false
```

Comment:
Is not `SINGLE_SHOT_OPERATIONS[0]` undefined?
Maybe you want to use `SINGLE_SHOT_OPERATIONS. REPORT_VULNERABILITY`